### DISCUSSION TITLE
Serene Latest release 8.6

### DISCUSSION THREAD

**Discussion by nahidulislam-cse15** [serenity-is/Serenity#7257] [Help] [answered]
When i run the project it generated code in server typing "serenity.corelib";
Ex:LanguageFor,.ts
import { PrefixedContext, initFormType } from "@serenity-is/corelib";
import { StringEditor } from "serenity.corelib";
how to solve this ?

**Comment by VictorTomaili** [comment, marked as answer]
update the sergen tool, delete the node modules and restore it back, delete bin folder and rebuild you app. Check the npm  packages and the nuget packages are latest.

**Comment by nahidulislam-cse15** [comment]
thanks
